README.md

```diff
@@ -18,9 +18,14 @@ This layer depends on:
 * poky (http://git.yoctoproject.org/cgit/cgit.cgi/poky)
 * meta-openembedded (http://cgit.openembedded.org/meta-openembedded/)
 * meta-virtualization (http://git.yoctoproject.org/cgit/cgit.cgi/meta-virtualization),
-if the "process-containment" distro feature has been enabled
+  if the "process-containment" distro feature has been enabled
 * meta-bistro (https://github.com/Pelagicore/meta-bistro)
 
+The relationship with meta-bistro, which is also developed at Pelagicore, is
```

Comment:
Should the layer name be in back ticks (`meta-bistro`)? Have to check how its done in the other parts of the SWF.